concepticondata/conceptlists/Verheyen-2019-1000.tsv

```diff
@@ -0,0 +1,1001 @@
+Verheyen-2019-1000-88	88	bewust	aware			8.85	4.5	3.6	4.6	5.7	3.8	4.75	6	2	30945	1	6003	862	1.14033314845811	0.000284021794692
+Verheyen-2019-1000-89	89	bezet	busy	BUSY	2755	6.2	4.2	5.1	4.5	4.95	4.8	3.4	5	2	21122	5	563	201	0.395435431380257	0.000107095146179
+Verheyen-2019-1000-90	90	bezig	busy	BUSY	2755	4.6	5.55	4.65	4.55	5.75	4.65	4.05	5	2	19105	7	7954	8602	1.13103032398977	0.000169204473434
+Verheyen-2019-1000-91	91	bezitterig	possessive			9.3	5.3	4.25	5.4	3.8	3.8	2.05	10	4	99813	0	0	36	0.259325346397403	3.52102546840378E-05
+Verheyen-2019-1000-92	92	bezorgd	afraid			7.55	4.75	4.35	3.65	5.25	5.05	4.15	7	2	23651	0	1037	1976	0.767920078549726	0.000127489107926
+Verheyen-2019-1000-93	93	Bijbels	biblical			7.15	3	4.35	3.4	1.95	4.2	3.1	7	2	43611	0	443	151	0.08352531429712	1.42663360937913E-05
+Verheyen-2019-1000-94	94	bijdehand	bright	BRIGHT	679	11.75	4.5	3.6	4.78947368421053	3.25	2.9	3.15	9	3	59264	0	26	236	0.198328174790019	4.32350687367953E-05
+Verheyen-2019-1000-95	95	bijzonder	particular			6.75	5.3	4.9	5.5	4.35	3.95	5.95	9	3	91367	0	10205	3719	1.33839888472491	0.000127339415531
+Verheyen-2019-1000-96	96	bitsig	bitter			11.375	5.14285714285714	3.66666666666667	5.375	2.25	4.1875	2.85	6	2	21976	2	32	2	0.146617215408819	8.68215891839304E-06
+Verheyen-2019-1000-97	97	bitter	bitter	BITTER	887	8.1	3.55	4.4	4.3	3.95	4.6	2.55	6	2	70206	8	1784	381	1.97431824661641	0.00039288396197
+Verheyen-2019-1000-98	98	bizar	bizarre			9.7	4.3	3.45	4.7	3.8	3.5	3.9	5	2	15065	1	334	583	0.609397407157788	0.000119073791973
+Verheyen-2019-1000-99	99	blank	blank			6.15	3.5	5.45	4.55	5.2	6.8	4.26315789473684	5	1	28609	5	967	3222	0.763803687425385	0.000142725190405
+Verheyen-2019-1000-100	100	blauw	blue	BLUE	837	2.8	3.05	6.3	3.9	6.05	6.7	4.9	5	1	14761	2	5281	3589	13.1504305102398	0.001937572804198
+Verheyen-2019-1000-101	101	bleek	pale			6.7	2.6	5.05	2.9	4.4	6.5	2.35	5	1	29523	2	1447	156	1.37810028519608	0.000210334086091
+Verheyen-2019-1000-102	102	blij	cheerful			3.6	5.85	3.95	4.6	6.05	6.15	6.2	4	1	24286	3	4391	12345	13.7477807648965	0.000940374134664
+Verheyen-2019-1000-103	103	blind	blind	BLIND	1653	6.75	3.3	6.35	3.15	5.25	5.45	2.05	5	1	43485	2	1207	1957	1.49027716107327	0.000269010250805
+Verheyen-2019-1000-104	104	bloedig	bloody			7.95	5.41176470588235	4.5	4.73684210526316	3.88888888888889	5.3	2.25	7	2	34622	1	336	120	0.012540759893701	6.35216424614064E-06
+Verheyen-2019-1000-105	105	bloedrood	blood-red			7.45	5	5.55	5.45	2.1	6.4	3.15	9	2	46968	0	147	20	0.030540072986547	4.04169466890711E-06
+Verheyen-2019-1000-106	106	blond	blond			6.45	4.2	6.35	3.7	5.25	6.8	4.35	5	1	30121	2	1291	999	2.49147921810933	0.000284770256668
+Verheyen-2019-1000-107	107	bloot	bare			5.05	4.65	5.75	3.7	5.1	6.4	4.8	5	1	21547	4	2435	775	2.21707634362985	0.000220223546718
+Verheyen-2019-1000-108	108	blut	flat	FLAT	1633	9.75	3.2	6.05	2.45	4.75	4.8	1.9	4	1	6543	5	46	546	0.457809567585147	6.12502230742106E-05
```

Comment:
not quite sure about blut, it seems to be colloquial and possibly denoting sth. like broke, without money